docs/canary-checker/docs/concepts/metrics.md

```diff
@@ -11,6 +11,91 @@ Canary checker comes with a built-in dashboard for displaying canary results, it
 
 ![](https://github.com/flanksource/docs/blob/85bdd4875d0d3ded16b7aa6c132d423852fcad90/docs/images/dashboard-http-pass-canary.png?raw=true)
 
+### Custom Metrics
+
+Export custom prometheus metrics from your check results
```

Comment:
Start with a brief explanation of what you can achieve e.g. replacing prometheus exporters / using CRD's for exporter config instread of configmaps.

Then example, then specs